src/Symfony/Bridge/Twig/composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/asset": "~3.4|~4.0",
         "symfony/dependency-injection": "~3.4|~4.0",
         "symfony/finder": "~3.4|~4.0",
-        "symfony/form": "^4.3.4",
+        "symfony/form": "^4.3.9",
```

Comment:
Why is that needed?

Reply:
My bad, 4.3.5 is enough, see https://github.com/symfony/symfony/pull/33573#discussion_r329006002